src/Field/Field.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import styled from 'styled-components'
+import { darken } from 'polished'
+
+import { Icon } from '../Icon'
+import { Text } from '../Text'
+import { Box } from '../Box'
+import { theme } from '../theme'
+
+type Props = {
+  children: React.ReactElement
+  showCaret?: boolean
+  renderAsTitle?: boolean
+  className?: string
+  id: string
+  error?: boolean // nb this needs to be optional for legacy support - but we will want to enforce in  the future
+  label?: string
+  outlined?: boolean
+  value: string
+  trailingIcon?: string
+  errorMsg?: string
+  dropdownKey?: string // nb only used by dropdown component but required on content in that context
+  required?: boolean // also optional for now to maintain backwards compaibility
```

Comment:
what do you mean by 'for now'? is there plans to change the way this is implemented?

Reply:
good spot, my bad - I'll remove the comment